csharp/Unnecessary.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using ShittyLINQ;
+using System.Collections.Generic;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        var integers = Enumerable.Range(1, 100);
+
+        foreach (var integer in integers)
+        {
+            new Unnecessary(integer);
+        }
+        Console.ReadLine();
+    }
+}
+
+public class Unnecessary
+{
+    
+    private static int _number;
+    
+    Func<int, int> GetFizzBuzz = number => number % 3 == 0 ? 0 : number % 5 == 0 ? 1 : -1;
```

Comment:
:throws resume in discard bin:
FizzBuzz solutions should print "FizzBuzz" when the number is divisible by both 5 and 3.
I would appreciate another nested ternary here.